source/SpinalHDL/Structuring/parametrization.rst

```diff
@@ -21,17 +21,19 @@ Here is an example of class parameters
 
 .. code-block:: scala
 
-  case class MyBus(width : Int) extends Bundle{
+  case class MyBus(width : Int) extends Bundle {
     val mySignal = UInt(width bits)
   }  
   
 .. code-block:: scala
 
-  case class MyComponent(width : Int) extends Component{
+  case class MyComponent(width : Int) extends Component {
     val bus = MyBus(width)
   }
   
-You can also use global variable defined in scala object's, but note that recently was added the ScopeProperty feature which improve on that solution.
+You can also use global variables defined in scala object's (which make them static like a
```

Comment:
```suggestion
You can also use global variables defined in scala objects (which make them static like a
```

Reply:
This file's changed are moved to other issue commits.